### PR TITLE
refine doc of getUrlWithUuid

### PR DESCRIPTION
**Pull request by SantyWang**
Re: cocos-creator/2d-tasks#

Changes:
 *

### CODE REVIEW

File: cocos2d/core/asset-manager/helper.js [hunk 1]
```diff
@@ -95,11 +95,17 @@ var helper = {
      * @method getUrlWithUuid
      * @param {string} uuid - The uuid of asset
      * @param {Object} [options] - Some optional parameters
+     * @param {Boolean} [options.isNative] - Indicates whether the path you want is a native resource path
+     * @param {string} [options.ext] - Extension of the resource path, it is required when isNative is true
```
Comment (jareguo) [inline cocos2d/core/asset-manager/helper.js]: 既然这样，我觉得应该是 options.nativeExt 才对

Comment (SantyWang) [reply cocos2d/core/asset-manager/helper.js]: 也可以

File: cocos2d/core/CCNode.js [hunk 2]
```diff
@@ -2767,6 +2767,8 @@ let NodeDefines = {
      * !#en
      * Sets the scale of axis in local coordinates of the node.
      * You can operate 2 axis in 2D node, and 3 axis in 3D node.
+     * When only (x, y) is passed, the value of scale.z will not be changed.
+     * When Vec2 is passed, the value of scale.z will be set to 0.
```
Comment (jareguo) [inline cocos2d/core/CCNode.js]: When a Vec2 is passed in, ...

File: cocos2d/core/CCNode.js [hunk 3]
```diff
@@ -2683,8 +2683,8 @@ let NodeDefines = {
     /**
      * !#en
      * Sets the position (x, y, z) of the node in its parent's coordinates.<br/>
-     * Usually we use cc.v2(x, y) to compose cc.Vec2 object,<br/>
-     * and passing two numbers (x, y) is more efficient than passing cc.Vec2 object.
+     * Usually we use cc.v2(x, y) to compose cc.Vec2 object,(position.z will become 0)<br/>
```
Comment (jareguo) [inline cocos2d/core/CCNode.js]: , in this case, position.z will become 0.

File: cocos2d/core/CCNode.js [hunk 4]
```diff
@@ -2683,8 +2683,8 @@ let NodeDefines = {
...
+     * and passing two numbers (x, y) is more efficient than passing cc.Vec2 object,(position.z will not change)
```
Comment (jareguo) [inline cocos2d/core/CCNode.js]: , in this case, position.z will remain unchanged

**Pull request merged**